libraries/chain/db_maint.cpp

```diff
@@ -778,20 +778,19 @@ void database::process_bitassets()
 
             const auto &asset = get(o.asset_id);
             auto flags = asset.options.flags;
-            if ((flags & witness_fed_asset) || (flags & committee_fed_asset)) // if smartcoin
+            if ( (flags & witness_fed_asset || flags & committee_fed_asset) &&
```

Comment:
To make sure `&` will be done before `||`, it's not harm to leave the `()`s.
BTW as @jmjatlanta mentioned, perhaps better to use `( flag & ( witness_fed_asset | committee_fed_asset ) )`